Makefile.toml

```diff
@@ -1056,3 +1056,14 @@ set -e
 
 cp -a .risingwave/rw-docker/* docker
 '''
+
+[tasks.generate-example-config]
```

Comment:
How about checking whether the `toml` is the latest on CI?

Reply:
Do you mean the compilation speed? We can move config to a separate crate 😇